standards/certification/overview.md

```diff
@@ -0,0 +1,9 @@
+# Overview
+
+SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate subjects, which are described further below.
```

Comment:
We decided to rephrase `certificate subject` to `certificate scope`, as decided in https://github.com/SovereignCloudStack/standards/issues/347